Modules/_sqlite/connection.c

```diff
@@ -893,7 +895,8 @@ print_or_clear_traceback(callback_context *ctx)
     assert(ctx != NULL);
     assert(ctx->state != NULL);
     if (ctx->state->enable_callback_tracebacks) {
-        PyErr_WriteUnraisable(ctx->callable);
+        PyErr_FormatUnraisable("Exception ignored on sqlite callable %R",
```

Comment:
It's either a callback (authorizer, progress, or trace callback), or a UDF (user-defined SQL function). Perhaps just go with "sqlite3 callback".

```suggestion
        PyErr_FormatUnraisable("Exception ignored on sqlite3 callback %R",
```